src/components/ContextMenuItem.tsx

```diff
@@ -66,12 +59,13 @@ function ContextMenuItem({onPress, successIcon, successText, icon, text, isMini,
 
         // We only set the success state when we have icon or text to represent the success state
         // We may want to replace this check by checking the Result from OnPress Callback in future.
-        if (successIcon || successText) {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
I guess it's not necessary, eslint shouldn't complain in this place
```suggestion
```

Reply:
Right, updated!